### PR TITLE
DO NOT MERGE YET: bugfix/crooked-navigator-outline

### PR DESCRIPTION
**Pull request by bm64**
Corrected crooked line in the navigator outline.

_Note to self_: Because of the large number of visual diff (mostly 1px), do not merge this until after the next release has been auto-visually tested, but before the manual tests.

### CODE REVIEW

**Review by pawelfus — APPROVED** [review]

**Comment by TorsteinHonsi** [comment]
Looks good, but the visual diff preview is down now so I can't verify.

File: samples/unit-tests/navigator/navigator/demo.js [hunk 1]
```diff
@@ -1407,3 +1407,39 @@ QUnit.test('Navigator, testing method: getBaseSeriesMin', function (assert) {
...
+                }
+            ]
+        });
+
+        chart.xAxis[0].setExtremes(0, 5);
+
+        const outlinePathArray = chart.navigator.outline.pathArray;
+
+        assert.equal(
+            outlinePathArray[0][2], //Upper left of navigator outline
```
Comment (TorsteinHonsi) [inline samples/unit-tests/navigator/navigator/demo.js:1440]: Lint error
```suggestion
            outlinePathArray[0][2], // Upper left of navigator outline
```

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks!

Our test suite takes some time to run, mainly because of the time it takes to generate the charts for each test. Therefore we have a policy to try to reuse existing charts for new regression tests. Can you find a similar test and apply the assertion there instead of creating a whole new test with a stock chart?

**Comment by TorsteinHonsi** [comment]
Thanks! Can you take a look at the conflicts too?

**Review by pawelfus — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**